src/zc/buildout/tests.py

```diff
@@ -2408,12 +2408,12 @@ def wont_downgrade_due_to_prefer_final():
     ... parts =
     ... ''')
 
-    >>> [v] = [l.split('= >=', 1)[1].strip()
+    >>> [v] = [str(l.split('= >=', 1)[1].strip())
```

Comment:
Because I got a spurious failure when one was unicode and the other a str in Python 2,